Dockerfile

```diff
@@ -11,6 +11,7 @@ ENV POLICY_PATH="/project"
 RUN ARCH=$(uname -m) && curl -L https://github.com/open-policy-agent/conftest/releases/download/v"${conftest_version}"/conftest_"${conftest_version}"_Linux_"$ARCH".tar.gz | tar -xz --no-same-owner -C /usr/bin/ && \
     curl https://mirror.openshift.com/pub/openshift-v4/"$ARCH"/clients/ocp/stable/openshift-client-linux.tar.gz --output oc.tar.gz && tar -xzvf oc.tar.gz -C /usr/bin && rm oc.tar.gz && \
     curl -LO "https://github.com/bats-core/bats-core/archive/refs/tags/v$BATS_VERSION.tar.gz" && \
+    curl -L https://github.com/operator-framework/operator-registry/releases/download/v1.26.3/linux-amd64-opm > /usr/bin/opm && chmod +x /usr/bin/opm && \ 
```

Comment:
Can we have the version as an argument here, similar to the `conftest_version` and other examples in this Dockerfile?